test/mpi/runtests.in

```diff
@@ -1000,15 +1000,17 @@ sub CleanUpAfterRun {
     @stillRunning = &FindRunning( $programname );
 
     if ($#stillRunning > -1) {
-        print STDERR "Some programs ($programname) may still be running:\npids = ";
-        for (my $i=0; $i <= $#stillRunning; $i++ ) {
-            print STDERR $stillRunning[$i] . " ";
+        if ($verbose) {
```

Comment:
I think this was already merged in another PR. I'm surprised Github isn't showing a conflict, but maybe it can be removed in the next update?

Reply:
Yes. The other PR was cherry-picked from here. Git will silently drop duplicate/noop patches, therefore it does not get treated as conflict. A rebase will remove this commit.